docs/schemaform/form-config.md

```diff
@@ -51,6 +52,11 @@ Forms are created by creating a page that uses FormApp from the schemaform folde
       // JSON schema object for the page. Follows the JSON Schema format.
       schema: {
         type: 'object',
+        // In a schema's properties, there are references to definitions. For example:
+        //   "homePhone": { "$ref": "#/definitions/phone" }
+        // In the config file, the definition for phone would then need to be added into definitions
+        // for it to be parsed correctly and added to homePhone.
+        definitions: {},
```

Comment:
We forgot to discuss this yesterday, but one suggestion is to go ahead and merge, and then later I can take a pass at revising this page to separate out the config snippet and a backend schema snippet to show how we're sharing definitions across the FE and BE.